.breakage-allowlist

```diff
@@ -1 +1,4 @@
-Func CarPlayManagerDelegate.carPlayManagerDidCancelPreview(_:) has been added as a protocol requirement
\ No newline at end of file
+Func CarPlayManagerDelegate.carPlayManagerDidCancelPreview(_:) has been added as a protocol requirement
+EnumElement RoadName.name has been removed
+EnumElement RoadName.code has been removed
+Enum RoadName has been changed to a Struct
```

Comment:
Are these changes generally acceptable for a non-major release? 
Do we get OK with these changes only because `RoadName` is expected to be used only in EV that is in public beta?